src/tests/integration/test-data/show/AssumeTest.testFail_assume_true(uint256,uint256).expected

```diff
@@ -5,64 +5,57 @@
 │   callDepth: 0
 │   statusCode: STATUSCODE:StatusCode
 │
-│  (856 steps)
+│  (547 steps)
 ├─ 3
-│   k: STATICCALL ( VGAS:Int +Int -3019 ) 645326474426547203313410069153905908525362434 ...
+│   k: STATICCALL 0 645326474426547203313410069153905908525362434349 128 36 128 0 ~> #p ...
 │   pc: 563
 │   callDepth: 0
 │   statusCode: STATUSCODE:StatusCode
 │
 │  (1 step)
 ├─ 4
-│   k: #checkCall 728815563385977040452943777879061427756277306518 0 ~> #call 728815563 ...
+│   k: #accessAccounts 645326474426547203313410069153905908525362434349 ~> #checkCall 7 ...
 │   pc: 563
 │   callDepth: 0
 │   statusCode: STATUSCODE:StatusCode
 │
 │  (1000 steps)
 ├─ 5
-│   k: #gas [ MSTORE , MSTORE 192 46308022326495007027972728677917914892729792999299745 ...
-│   pc: 3667
+│   k: #addr [ ISZERO ] ~> #exec [ ISZERO ] ~> #pc [ ISZERO ] ~> #execute ~> CONTINUATI ...
+│   pc: 4206
 │   callDepth: 0
 │   statusCode: STATUSCODE:StatusCode
 │
-│  (1000 steps)
+│  (827 steps)
```

Comment:
This change in the resulting KCFG is concerning, and should probably be isolated into a regression test for the booster backend.

Reply:
I've tested locally and it turns out that the booster and legacy backends actually produce the same output on this test. This test wasn't running before (being used for any output), because it's listed in the tests for the legacy backend to skip, and so was being skipped for the show step as well.